src/instructlab/eval/mmlu.py

```diff
@@ -30,10 +36,49 @@ def run(self) -> tuple:
             overall_score       MMLU score for the overall model evaluation
             individual_scores   Individual MMLU score for each task
         """
-        individual_scores: dict[str, float] = {}
-        overall_score: float = 0.0
+        #TODO: make this a parameter for class?
+        os.environ['TOKENIZERS_PARALLELISM'] = 'true'
+
+        individual_scores: dict = {}
+        agg_score: float = 0.0
+        model_args = "pretrained=" + self.model_path + ",dtype=" + self.model_dtype
+
+        mmlu_output = simple_evaluate(
+            model="hf",
+            model_args=model_args,
+            tasks=self.tasks,
+            num_fewshot=self.few_shots,
+            batch_size=self.batch_size
+        )
+
+        results = mmlu_output["results"]
+
+        for task in self.tasks:
+            mmlu_res = results[task]
+            agg_score += float(mmlu_res['acc,none'])
+            individual_scores[task] = {}
+            individual_scores[task]['score'] = float(mmlu_res['acc,none'])
+            individual_scores[task]['stderr'] = float(mmlu_res['acc_stderr,none'])
+
+        overall_score = float(agg_score/len(self.tasks))
         return overall_score, individual_scores
+    
+
+############# Testing Code Follows ##############
+def main():
+    # Path to the granite model in the aliryan vm on AWS
+    model_path = "/home/ec2-user/instructlab/models/instructlab/granite-7b-lab"
+    #TODO: all 57 tasks need to be parameterized possibly by CLI
+    tasks = ["mmlu_abstract_algebra","mmlu_anatomy","mmlu_astronomy"]
+    dtype = "float16"
+    mmlu = MMLU_Evaluator(model_path, tasks, dtype, 2, 5)
+    overall_score, individual_scores = mmlu.run()
```

Comment:
I changed run to more specific methods in #9.  We don't have to here but it could leave room for other functions in the future.  Using run seems to imply the object really only does 1 thing.

Reply:
That makes sense, I am going to keep it to run for now and could change this in a follow-up PR.

Reply:
Added follow-up issue here: https://github.com/instructlab/eval/issues/16